package.json

```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "broccoli-funnel": "2.0.1",
     "ember-cli-babel": "6.8.2",
-    "rollbar": "~2.2.7"
+    "rollbar": "~2.2.8"
```

Comment:
We don't have to bump it as we have `~` which will automatically resolve `rollbar` to the latest `2.2.*` version

Reply:
It still installed 2.2.7 this morning. It installs 2.2.8 now, so thanks!